sentinel-core/src/main/java/com/alibaba/csp/sentinel/config/SentinelConfigLocator.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.csp.sentinel.config;
+
+import com.alibaba.csp.sentinel.log.RecordLog;
+import com.alibaba.csp.sentinel.util.AppNameUtil;
+import com.alibaba.csp.sentinel.util.ConfigUtil;
+import com.alibaba.csp.sentinel.util.StringUtil;
+
+import java.io.File;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.CopyOnWriteArraySet;
+
+import static com.alibaba.csp.sentinel.util.ConfigUtil.addSeparator;
+
+/**
+ * <p>
+ * class responsible for loading the Sentinel Core configuration
+ * </p>
+ *
+ * @author lianglin
+ * @since 1.7.0
+ */
+public class SentinelConfigLocator {
+
+
+    private static final String DIR_NAME = "logs" + File.separator + "csp";
+    private static final String USER_HOME = "user.home";
+
+    private static String SENTINEL_CONFIG = "csp.sentinel.config.file";
```

Comment:
Maybe this key can be public?

Reply:
I make `SENTINEL_CONFIG` private modified , because unit  now it only be used in `SentinelConfigLocator.java`.  If it is accessed anywhere else in the future, it really needs to be public. Ok, I can change it into public and modify it with final

Reply:
Each config key constant could be public.

Reply:
emm, I had committed latest code. Could you help me view it?

Reply:
File.separator rather than "/"